lucille-core/src/test/java/com/kmwllc/lucille/stage/ExtractEntitiesAhoTest.java

```diff
@@ -0,0 +1,28 @@
+package com.kmwllc.lucille.stage;
+
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigValueFactory;
+
+import java.lang.reflect.Constructor;
+
+public class ExtractEntitiesAhoTest extends ExtractEntitiesTest {
+
+  @Override
+  protected Stage newStage(String hoconBody) {
+    try {
+      Config cfg = ConfigFactory.parseString(hoconBody).resolve()
+          .withValue("class", ConfigValueFactory.fromAnyRef("com.kmwllc.lucille.stage.ExtractEntities"));
+      Class<? extends Stage> cls = com.kmwllc.lucille.stage.ExtractEntities.class;
```

Comment:
You don't need to invoke the constructor with reflection here (lines 18-20), you can do it directly as `new ExtractEntities(config)` because you know the one class you're trying to instantiate